### DISCUSSION TITLE
Filter commit by path

### DISCUSSION THREAD

**Discussion by bogaertg** [semantic-release/semantic-release#2316] [Ideas]
Hello,

I made a pull request to add the possibility to filter commit by path
Does someone can review my [pull request](https://github.com/semantic-release/semantic-release/pull/2315) ?

**Comment by bogaertg** [comment]
up ?
@travi @gr2m

**Comment by gr2m** [comment]
the link to the pull request is not right.

But please do not double ping us. We get to it when we get to it.

**Reply by bogaertg** [reply]
Sorry for the double ping
This feature is important for my work.
the link is fixed 👍

**Reply by bogaertg** [reply]
any news ?
